.github/workflows/workflow.yml

```diff
@@ -144,19 +146,18 @@ jobs:
           path: .
       - name: S3 Upload basilisk-testnet-chainspec.json
         uses: tpaschalis/s3-sync-action@master
-
   destroy-runner-instance:
     name: destroy runner
     runs-on: ubuntu-latest
-    needs: [pop-runner-instance, build, build-and-publish-docker-image]
+    needs: pop-runner-instance
```

Comment:
this will imidiatelly destroy runner after creating it, not sure if we want that